src/components/DayPickerRangeController.jsx

```diff
@@ -107,6 +107,7 @@ export default class DayPickerRangeController extends React.Component {
     };
 
     this.isTouchDevice = isTouchDevice();
+    this.today = moment();
```

Comment:
I have a concern about the datepicker being open for a while, or someone using it across day boundaries. Let's leave it in the constructor but also maybe update it in `componentWillUpdate` (reset it as `this.today = moment()` again there.